src/@types/types.d.ts

```diff
@@ -53,12 +54,13 @@ export type NavItem = {
   isHidden?: boolean;
   permissions?: NavItemPermission[] | NavItemPermission;
   dynamicNav?: string;
+  description?: string;
 };
 
 export type BundleNavigation = {
   id: string;
   title: string;
-  navItems: NavItem[];
+  navItems?: NavItem[];
```

Comment:
This does not have to be optional. Instead, we should use the `BundleNavigation` type for the first axios call in `useAllServices` `fetchNavitation` callback. We currently use `BundleNav`. But the fallback uses `BundleNavigation` anyways,